games/y.yaml

```
@@ -5,9 +5,9 @@
   development: active
   originals:
   - Quake 2
-  repo: https://github.com/yquake2
+  repo: https://github.com/yquake2/yquake2
```

```suggestion
  repo: https://github.com/yquake2
```
let's leave this unchanged; if dangerJS is broken it should be fixed instead